pints/_nested/__init__.py

```diff
@@ -414,6 +419,8 @@ def _initialise_logger(self):
             self._logger.add_time('Time m:s')
             self._logger.add_float('Delta_log(z)')
             self._logger.add_float('Acceptance rate')
+            if self._sampler._multiple_ellipsoids:
```

Comment:
In the other controllers we handle this in a more general way, by letting each sampler/optimiser define a method _log_init (that does nothing by default) which can update the logging

https://github.com/pints-team/pints/blob/master/pints/_mcmc/__init__.py#L623-L624

and `_log_write`:

https://github.com/pints-team/pints/blob/master/pints/_mcmc/__init__.py#L820-L821